File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { BrontieSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = BrontieSDK.test()
    equal(testsdk instanceof BrontieSDK, true,
      'BrontieSDK.test() must return a client synchronously')
  })

})

File: lua/test/exists_test.lua
-- Brontie SDK exists test

local sdk = require("brontie_sdk")

describe("BrontieSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
